### DISCUSSION TITLE
Versione sqllite

### DISCUSSION THREAD

**Discussion by galtamur** [indomus/forum#1076] [HUB: Home Assistant]
Ciao a tutti, dopo l'installazione di HA 2021.11, ricevo nel log, alla partenza di HA, questo messaggio:
Version 3.27.2 of SQLite is not supported; minimum supported version is 3.31.0. Starting with Home Assistant 2022.2 this will prevent the recorder from starting. Please upgrade your database software before then

Solo io ho questo problema?
Qualcuno sa indicarmi come eseguire l'upgrade di SQLite?

**Comment by snakuzzo** [comment]
No...ce l'hanno tutti quelli che usano HA su venv su Buster.
Dovresti aggiornare il motore sqlite, ma tramite apt coi repository di buster, non trova la versione ultima che ti occorre. Quindi, premesso che hai ancora tempo fino a febbraio per vedere se verrà aggiunto, puoi comunque procedere alla compilazione dai sorgenti come indicato [QUI](https://github.com/indomus/forum/discussions/1016#discussion-3672693)
